pfcpiface/parse-pdr.go

```diff
@@ -142,9 +142,6 @@ func (p *pdr) parsePDI(pdiIEs []*ie.IE, appPFDs map[string]appPFD, upf *upf) err
 		if p.srcIface == core {
 			p.dstIP = ip2int(ueIP4)
 			p.dstIPMask = 0xffffffff // /32
-		} else if p.srcIface == access {
-			p.srcIP = ip2int(ueIP4)
-			p.srcIPMask = 0xffffffff // /32
```

Comment:
Can you explain whats happening here?

Reply:
In case of src interface being access.. If we receive UE IP in the PDR we set srcIP to be same as UE IP which is not correct. when we receive the packet the srcIP will be enbIP. so commented it for now. we can just compare the destination tunnel ip and port for the packets.

Reply:
Anything related to tunnel starts with `tunnel` in the PDR struct.
https://github.com/omec-project/upf-epc/blob/6060980b46840b048a83954685efb5040af272ab/pfcpiface/parse-pdr.go#L14-L21

Without that we are referring to the internet packet only. This is not describing P4 or in general packet processing. If thats confusing we can rename src and dest as UE and app which will put a little more work in some other places.